addons/website/static/src/js/website.contentMenu.js

```diff
@@ -12,7 +12,9 @@
             self.$el.on('click', 'a[data-action]', function(ev) {
                 ev.preventDefault();
                 var $content_item = $(this);
-                self[$content_item.data('action')]();
+                var data_action = self[$content_item.data('action')];
+                if (data_action)
+                    data_action();
```

Comment:
Is this change still required ? If averything works without it, remove it.
If you keep it, add '{}' toavoid error when minified.